### DISCUSSION TITLE
Cayman Theme not rendering from Demo

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22284] [Other Feature Feedback, Questions, & Ideas]
<p>I went through the GitPages demo this morning, didn’t see anything but a 404.  Now I went and changed the theme on the repo’s settings page and see the “hello World” text but not theme.  Are these pages lagging for some reason?</p>
<p>This is the demo I went through; pretty basic – <a href="https://pages.github.com/" rel="nofollow noopener">https://pages.github.com/</a> – Everything was working fine with non errors, except step 5.  Just a 404.  I changed the theme here – <a href="https://github.com/daveOrleans/daveOrleans.github.io/settings" rel="nofollow noopener">https://github.com/daveOrleans/daveOrleans.github.io/settings</a>  in the GitPages section and applied Cayman but only the old hello world text. Nothing else.   Thanks.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/4107d620c9f56ea10e9f7a475a57521bcfe51981.png" data-download-href="/uploads/short-url/9hhP9zel4mXj2UN8ZkbZmZbFxSx.png?dl=1" title="GitPages.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/5883591/181070059-b8e245ae-3d8b-43fc-b004-2025ff5e9af2.png" alt="GitPages.png" data-base62-sha1="9hhP9zel4mXj2UN8ZkbZmZbFxSx" width="690" height="314" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/4107d620c9f56ea10e9f7a475a57521bcfe51981_2_690x314.png, https://aws1.discourse-cdn.com/github/original/1X/4107d620c9f56ea10e9f7a475a57521bcfe51981.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/4107d620c9f56ea10e9f7a475a57521bcfe51981.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/4107d620c9f56ea10e9f7a475a57521bcfe51981_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">GitPages.png</span><span class="informations">1024×467 22.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by RDIL** [comment, marked as answer]
<p>Hey,<br>
The GitHub pages cayman theme only renders markdown files. You will need to delete your index.html and create a README.md, the content in there should be served.</p>
<p>Hope this helps!</p>
